Sourcery/Generating/Template/JavaScript/JavaScriptTemplate.swift

```diff
@@ -20,11 +20,34 @@ final class JavaScriptTemplate: Template {
         let path = Bundle(for: JavaScriptTemplate.self).path(forResource: "ejsbundle", ofType: "js")!
         let ejs = try String(contentsOfFile: path, encoding: .utf8)
 
+        var errorContent = ""
+
+        let include: @convention(block) (String) -> String? = { [unowned self] path in
+            let path = self.sourcePath.parent() + Path(path)
+            let includedTemplate: String? = try? path.read()
+            return includedTemplate
+        }
+
+        let rethrow: @convention(block) (NSDictionary, String, String, String) -> Void = { error, source, filename, lineno in
```

Comment:
You can set exception handler on the js context and throw error directly in js instead of calling a `rethrow` block. Also you will not need a try/catch. I think it will simplify things.